src/test/java/racing/domain/GameInfoTest.java

```diff
@@ -0,0 +1,44 @@
+package racing.domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import racing.input.InputValue;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class GameInfoTest {
+
+    @Test
+    @DisplayName("경기에 참여하는 차가 없을시 예외 발생 확인.")
+    public void hasCarTest() {
+
+        InputValue inputValue = new InputValue("0", "1");
+        Throwable exception = assertThrows(ArithmeticException.class, () -> {
+            new GameInfo(inputValue);
+        });
+
+        assertEquals("경주에 참여하는 자동차가 없습니다.", exception.getMessage());
```

Comment:
```suggestion
        assertThatThrownBy(() -> new GameInfo(inputValue))
                .isInstanceOf(ArithmeticException.class)
                .hasMessage("경주에 참여하는 자동차가 없습니다.");
```
`org.assertj.core.api.Assertions.assertThatThrownBy`를 이용한다면 위와 같이 바꿀수 있을것 같네요!